WordPress/Classes/Services/MediaCoordinator.swift

```diff
@@ -14,15 +14,12 @@ class MediaCoordinator: NSObject {
 
     private let queue = DispatchQueue(label: "org.wordpress.mediauploadcoordinator")
 
-    private lazy var mediaProgressCoordinator: MediaProgressCoordinator = {
+    private(set) lazy var mediaProgressCoordinator: MediaProgressCoordinator = {
```

Comment:
I'd prefer we keep the progress coordinator private, so that other classes using the `MediaCoordinator` don't have to know about it. It looks like Aztec is only using a few methods from the progress coordinator – for errors, `isRunning`, and accessing failed media. Can we expose those through a few small methods on `MediaCoordinator` instead?